website/docs/r/container_cluster.html.markdown

```diff
@@ -36,9 +36,7 @@ resource "google_container_cluster" "primary" {
   }
 
   node_config {
-    oauth_scopes = [
-      "https://www.googleapis.com/auth/compute",
-      "https://www.googleapis.com/auth/devstorage.read_only",
+    oauth_scopes = [ 
```

Comment:
Are those spaces at the end of line necessary?..

Reply:
Re-pushed with whitespace stripped